api_specifications/case-api-0.1.0.yaml

```
@@ -1499,3 +1554,178 @@ components:
+          description: The advocate ID associated with the case.
+        isReplacing:
+          type: boolean
+          description: Indicates whether the advocate is replacing an existing advocate or joining as an associate advocate
+        isJudgeApproving:
+          type: boolean
+          description: Indicates whether the judge is approving the joining of the advocate
+        reason:
+          type: string
+          description: Reason for joining the case
+        reasonDocument:
+            $ref: >-
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 12

(indentation)